Elizabeth_Theme/src/css/layout/_navigation.scss

```diff
@@ -0,0 +1,89 @@
+.theme-header {
+	background-color: #0000;
+	height: 12vh;
+	max-width: 100%;
+	position: fixed;
+	transition: 500ms ease;
+	width: 100vw;
+	z-index: 1;
+
+	.theme-user {
+		justify-content: flex-end;
+	}
+
+	.theme-user a {
+		color: $theme-main-color;
+	}
+
+	.user-avatar-link .lexicon-icon {
+		color: $theme-main-color;
+	}
+
+	li.nav-item {
+		a.nav-link {
+			color: #fff;
+		}
+
+		&:hover {
+			a {
+				color: $theme-main-color;
+			}
+
+			ul.child-menu {
+				display: block;
+				margin-top: -10px;
+			}
+		}
+
+		&.selected {
+			height: 73px;
+
+			& a.nav-link {
+				color: $theme-main-color;
+			}
+		}
+
+		@include media-breakpoint-down(sm) {
+			ul.child-menu {
+				display: block;
+			}
+		}    
+	}
+
+	li.selected {
+		& a.nav-link {
+			color: $theme-main-color;
+		}
+	}
+}
+
+.navbar-nav {
+  	justify-content: center;
+}
+
+.burger {
+  	display: none;
+}
+
+.burger div {
+	background-color: #fff;
+	height: 3px;
+	margin: 5px;
+	width: 25px;
+}
```

Comment:
If the `.burger` is set to `display: none;`,  then when would any of this CSS ever apply?